test/core_fusion.cpp

```diff
@@ -702,3 +702,93 @@ TEST(batch_fusion, pass_property)
     ASSERT_EQ(true, pass->get_property(pass::PassProperty::REQUIRE_STATIC_SHAPE));
     ASSERT_EQ(false, pass->get_property(pass::PassProperty::CHANGE_DYNAMIC_STATE));
 }
+
+#ifndef NGRAPH_JSON_DISABLE
```

Comment:
Is this defined or not defined by default?

Reply:
```
option(NGRAPH_FAST_MATH_ENABLE "Enable fast math" ON)
option(NGRAPH_JSON_ENABLE "Enable JSON based serialization and tracing features" TRUE)
```
its JOSN is enabled by default